tests/unittest/_torch/misc/test_autotuner.py

```
@@ -1127,3 +1127,134 @@ def forward(self,
+    exact winning tactic can differ. Only the achieved runtime is a meaningful
+    invariant, hence the tolerance comparisons.
+
+    Blackwell only (SM100/SM103) and requires the nvMatmulHeuristics library;
+    skipped otherwise.
+    """
+    if not torch.cuda.is_available():
+        pytest.skip("requires a CUDA device")
+    try:
+        from tensorrt_llm._utils import get_sm_version
+        sm_version = get_sm_version()
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)